prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py

```diff
@@ -133,11 +133,10 @@ def omen_replicate_from_tx(
             f"Created `https://aiomen.eth.limo/#/{market_address}` for `{market.question}` in category {category} out of {market.url}."
         )
 
-        omen_agent_market = OmenAgentMarket.from_data_model(
-            OmenSubgraphHandler().get_omen_market_by_market_id(market_address)
-        )
-
-        if generate_and_set_image_for_market(omen_agent_market, api_keys) is not None:
+        if (
```

Comment:
Because of the graph latency, sometimes it happens that the created market is not yet indexed in the graph and then replication fails, because `OmenSubgraphHandler().get_omen_market_by_market_id(market_address)` will error out.

So instead, provide just the address and question directly.